press/scripts/migrate.py

```diff
@@ -270,16 +272,18 @@ def select_site():
 			).strip()
 			if selected_site in available_sites:
 				site_data = available_sites[selected_site]
-				downgrade = is_downgrade(sites_info[selected_site])
-
-				if (not downgrade) or (
-					downgrade
-					and click.confirm(
-						"Downgrading may lead to a broken site. Are you sure you want to do this?"
-					)
-				):
+				global has_external_files
+				if not has_external_files:
+					downgrade = is_downgrade(sites_info[selected_site])
+					if (not downgrade) or (
```

Comment:
This probably can be a single `if` instead of being nested. Expression in the nested `if` is too long. Expressions should not span multiple lines.

Unnecessary indentation could be avoided by early exiting; shifting the else clause above the if.

https://gist.github.com/18alantom/d9f0565c0f42d6a71311d4a3093a1331#code-blocks